pkg/operator/ceph/csi/spec.go

```diff
@@ -498,6 +522,23 @@ func deleteCSIDriverResources(
 	return succeeded
 }
 
+func applyCephClusterNetworkConfig(objectMeta *metav1.ObjectMeta, rookclientset rookclient.Interface) (bool, error) {
+	cephClusters, err := rookclientset.CephV1().CephClusters(objectMeta.Namespace).List(metav1.ListOptions{})
```

Comment:
question: won't this fail if the ceph cluster and CSI is in a different namespace?

Reply:
yes it will fail

Reply:
Yes. 
The namespace we have here is actually the Rook Ceph operators namespace, right?

Reply:
Yes its rook ceph operator namespace